src/test/resources/application.yaml

```diff
@@ -0,0 +1,26 @@
+server:
+  port: 9001
+  contextPath:
```

Comment:
Property 'server.context-path' is Deprecated: Use 'server.servlet.context-path' instead.